runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaCacheServerConsumerFactory.java

```diff
@@ -578,7 +582,8 @@ private KafkaCacheServerConsumerFanout(
             this.groupId = groupId;
             this.timeout = timeout;
             this.streams = new ArrayList<>();
-            this.members = new Object2ObjectHashMap<>();
+            this.tempMembers = new ObjectHashSet<>();
+            this.members = new LinkedHashMap<>();
```

Comment:
Are we switching to `LinkedHashMap` to maintain insertion order, or for `putIfAbsent`?
Note: putIfAbsent is also available on `Object2ObjectHashMap`, inherited from `Map`.

Reply:
to maintain insertion order